worlds/witness/__init__.py

```diff
@@ -129,18 +128,18 @@ def determine_sufficient_progression(self):
                             f" progression items that can be placed in other players' worlds. Please turn on Symbol"
                             f" Shuffle, Door Shuffle or Obelisk Keys.")
```

Comment:
```suggestion
                            f" Shuffle, Door Shuffle, or Obelisk Keys.")
```
This was close enough to changes that I could suggest it. Oxford comma my beloved